samples/toolbarconfigurator/js/toolbarmodifier.js

```diff
@@ -260,7 +260,7 @@
 		this.modifyContainer.addClass( 'hidden' );
 		this.configContainer.removeClass( 'hidden' );
 
-		this.configContainer.setHtml( content );
+		this.configContainer.setHtml( CKEDITOR.tools.htmlSafeByReview(content, 'Content created inline using safe internal values. cfg is set to internal config values then its values are pulled for the template.') );
```

Comment:
SG!

Reply:
Was this removed too?

Reply:
Yep, samples/ doesn't look like it ends up in the build file (since they're just samples) so there's not much point making those changes.